test/google_search_data_viewer/accounts_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule GoogleSearchDataViewer.AccountsTest do
+  use GoogleSearchDataViewer.DataCase
+
+  alias GoogleSearchDataViewer.Accounts
+
+  describe "users" do
+    alias GoogleSearchDataViewer.Accounts.User
+
+    import GoogleSearchDataViewer.AccountsFixtures
+
+    @invalid_attrs %{email: "someemail", password: "somepassword"}
+    @valid_email "test@gmail.com"
+    @valid_password "aValidPasswordEntered"
+
+    test "list_users/0 returns all users" do
+      user = user_fixture()
+      assert Accounts.list_users() == [user]
+    end
```

Comment:
Let wrap each method using `describe` block with method name and arity.

```suggestion
describe "list_users/0" do
    test "returns all users" do
      user = user_fixture()
      assert Accounts.list_users() == [user]
    end
 end
```

Reply:
Thanks, done.